### PR TITLE
regression test for VMS-2231 standard priority updates missing

### PR DESCRIPTION
**Pull request by makarandp0**
This adds some integration tests that verify that when publisher changes priority from a->b, 
subscriber gets notified of the change. Currently they are failing when b=standard.

TODO:
- [x] remove console.logs
- [x] merge when [VMS-2231](https://issues.corp.twilio.com/browse/VMS-2231) is fixed

**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

File: test/integration/spec/localtrackpublication.js [hunk 1]
```diff
@@ -407,6 +405,55 @@ describe('LocalTrackPublication', function() {
       });
     });
 
+
+    [PRIORITY_LOW, PRIORITY_STANDARD, PRIORITY_HIGH].forEach(beforePriority => {
+      [PRIORITY_LOW, PRIORITY_STANDARD, PRIORITY_HIGH].forEach(afterPriority => {
+        if (beforePriority !== afterPriority) {
```
Comment (syerrapragada) [inline test/integration/spec/localtrackpublication.js]: We could remove this check here and have a test for updating the same priority as before. WDYT?

Comment (makarandp0) [reply test/integration/spec/localtrackpublication.js]: if priority does not change then we won't receive `publishPriorityChanged`

Comment (syerrapragada) [reply test/integration/spec/localtrackpublication.js]: Yes, we could have that check too in the test?

Comment (makarandp0) [reply test/integration/spec/localtrackpublication.js]: Thanks for the suggestion - I have updated the test to handle this case.

**Pull request merged**